src/tab-container-element.ts

```diff
@@ -160,7 +167,17 @@ export class TabContainerElement extends HTMLElement {
         this.#tabListSlot.assign(...[...this.children].filter(e => e.matches('[role=tab]')))
         this.#tabListSlot.role = 'tablist'
       }
+      const tabList = this.#tabList
+      if (this.hasAttribute('aria-description')) {
```

Comment:
It's only used twice but we could refactor this logic to save a few lines of code maybe? Not a blocker!